anx/provider/loadbalancer/address/prefix.go

```diff
@@ -36,6 +41,20 @@ func newPrefix(ctx context.Context, client anxprefix.API, identifier string) (*p
 		ret.family = v1.IPv4Protocol
 	}
 
+	if autoDiscoveryName != nil {
+		tag := fmt.Sprintf("kubernetes-lb-vip-%s", *autoDiscoveryName)
+		vip, err := ret.discoverVIP(ctx, apiclient, ipamClient, tag)
+		if err != nil {
```

Comment:
it might be good to differentiate between errors and "no error, but also no VIP found" - only falling back to calculated VIP in the second case